.eslintrc.js

```diff
@@ -1,3 +1,10 @@
 const { createConfig } = require('@edx/frontend-build');
 
-module.exports = createConfig('eslint');
+module.exports = createConfig('eslint',
+  {
+    'rules': {
+      'import/no-extraneous-dependencies': 'off',
```

Comment:
I've added this since otherwise `rosie`, which is a devDependency is incorrectly recommended as a regular dependency since it's imported in a non-test file. This can be fixed in `edx-lint`.